odl/discr/lp_discr.py

```diff
@@ -17,10 +17,6 @@
 
 """:math:`L^p` type discretizations of function spaces."""
 
-# TODO: write some introduction doc
```

Comment:
There is a math-style introduction, but implementation guide is still missing. I'll make an issue.